roxctl/declarativeconfig/lint/lint.go

```diff
@@ -30,31 +31,57 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 
 	cmd.Flags().StringVarP(&lintCmd.file, "file", "f", "", "file containing the declarative configuration in YAML format")
 
-	utils.Must(cmd.MarkFlagRequired("file"))
+	cmd.Flags().String(configmap.ConfigMapFlag, "", `config map from which to read the declarative configuration from.
+In case this is not set, the declarative configuration will be read from the YAML file provided via the --file flag.`)
+	cmd.Flags().String(configmap.NamespaceFlag, "", `namespace of the config map from which to read the declarative configuration from.
+In case this is not set, the namespace set within the current kube config context will be used`)
+
+	cmd.MarkFlagsMutuallyExclusive("file", configmap.ConfigMapFlag)
+
+	flags.HideInheritedFlags(cmd)
+
 	return cmd
 }
 
 type lintCmd struct {
 	env environment.Environment
 
 	file         string
-	fileContents []byte
+	fileContents [][]byte
```

Comment:
why this change? Do we allow content of multiple files?

Reply:
Change was mostly done to only have a single variable. The config map can have multiple file contents, while the `--file` flag will only yield one. So, essentially this was done to avoid having one for the file input, and one for the config map input.